atom/browser/atom_browser_main_parts.cc

```diff
@@ -550,7 +550,7 @@ void AtomBrowserMainParts::PreMainMessageLoopStartCommon() {
 
 #if defined(OS_MACOSX)
   if (custom_locale_.empty())
-    l10n_util::OverrideLocaleWithCocoaLocale();
+    custom_locale_ = GetSystemPreferredLanguage();
```

Comment:
are you sure we don't still need to call `OverrideLocaleWithCocoaLocale`? Also, do we need to check that the system's preferred language is one that we have localization resources for?

Reply:
I think the app should do that, and then choose the next one from the list of preferred languages.